include/seqan3/range/container/bitcompressed_vector.hpp

```diff
@@ -65,7 +66,7 @@ class bitcompressed_vector
 {
 private:
     //!\brief The number of bits needed to represent a single letter of the alphabet_type.
-    static constexpr size_t bits_per_letter = std::ceil(std::log2(alphabet_size<alphabet_type>));
+    static constexpr size_t bits_per_letter = detail::next_power_of_two(alphabet_size<alphabet_type>);
```

Comment:
```suggestion
    static constexpr size_t bits_per_letter = detail::most_significant_bit_set(detail::next_power_of_two(alphabet_size<alphabet_type>));
```

Should be the way. Can you check intendation?

Reply:
Hmm `next_power_of_two` says in the documentation: Returns 2^{\lceil\log_2(n)\rceil}. Ah we want that without the 2^. 

Shouldn't be detail::most_significant_bit_set enough then?

Maybe we should add the description that it also gives the `ceil(log_2(n))`

```suggestion
    static constexpr size_t bits_per_letter = detail::most_significant_bit_set(alphabet_size<alphabet_type>);
```

Reply:
Not really

E.g.:

```
62
0011 1110
```

MSB is 5, we need 6

```
64
0100 0000
```

MSB is 6, but we also just need 6 bit